jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -186,6 +186,46 @@ def parameterized(harness_group: Iterable[Harness],
   ]
 )
 
+
+lax_sort = jtu.cases_from_list( # one array, random data, all axes, all dtypes
```

Comment:
I have just discovered while working on #3700 that jtu.cases_from_list may drop test cases if JAX_NUM_GENERATED_TESTS is too small. Let's use `tuple` instead